src/APL/firestore-apl.test.ts

```diff
@@ -0,0 +1,72 @@
+import { CollectionReference, Firestore } from "@google-cloud/firestore";
+import { describe, expect, it } from "vitest";
+
+import { AuthData } from "./apl";
+import { FirestoreAPL } from "./firestore-apl";
+
+let firestore: Firestore | null = null;
+
+try {
+  firestore = new Firestore({
+    // @ts-ignore ignore import.meta, TODO
```

Comment:
I dont know how to set it. I dont want to mess with tsup, but also I can't configure it well. Changing modules in tsconfig either fails import.meta or fail deps imports.

@zaiste can you have a look, how should this be configured? Values *exist*, so probably we don't have to change tsconfig, but only add typings?